source/rebuild.js

```diff
@@ -0,0 +1,22 @@
+import _curry2 from './internal/_curry2.js';
+import _rebuild from './internal/_rebuild.js';
+
+/**
+ * Transforms an object into a new one, applying to every key-value pair a
+ * function creating zero, one, or many new key-value pairs, and combining
+ * the resulst into a single object.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig ([String, a] -> [[String, b]]) -> Object<a> -> Object<b>
```

Comment:
We don't use this `<>` in `Object<a>` syntax anywhere else. Maybe `{k: a}` and `{k: b}` or just `Object`?

Reply:
Right.  The stuff I meant to go back and fix before I submitted... ah well, it's been a while.